src/proposals/Proposal.sol

```diff
@@ -27,78 +27,81 @@ abstract contract Proposal is Test, Script, IProposal {
     /// they all follow the same structure
     Action[] public actions;
 
-    /// @notice debug flag to print proposal actions, calldata, new addresses and changed addresses
-    /// @dev default is true
-    bool internal DEBUG = false;
+    /// @notice debug flag to print internal proposal logs
+    bool internal DEBUG;
+    bool internal DO_DEPLOY;
+    bool internal DO_AFTER_DEPLOY_MOCK;
+    bool internal DO_BUILD;
+    bool internal DO_SIMULATE;
+    bool internal DO_VALIDATE;
+    bool internal DO_PRINT;
 
     /// @notice Addresses contract
     Addresses public addresses;
 
-    /// @notice the actions caller name in the Addresses JSON
-    string public caller;
```

Comment:
what are we deleting the caller. I think caller can be set at the most base level Proposal contract.

Reply:
because we don’t need it anymore, the caller was needed before because we were using the _outBuild function since we replaced with modifier, there is no need to have caller save to storage